pkg/monitor/intervalcreation/rendering.go

```diff
@@ -87,6 +88,9 @@ func BelongsInSpyglass(eventInterval monitorapi.EventInterval) bool {
 	if IsPodLifecycle(eventInterval) {
 		return false
 	}
+	if IsPathologicalEvent(eventInterval) {
+		return true
```

Comment:
this if/then is pointless because the function returns true anyway.  But I don't know why the events ending with `(n times)` are not showing up in the html/json files in the junit artifact dir.

This [job](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/27649/pull-ci-openshift-origin-master-e2e-aws-ovn-single-node-upgrade/1614985456145928192) had some patho failures but the events are not in the `e2e-timelines_kube-control-plane_20230116-145353.html` file as I would've expected.